extensions/wikia/EditPageLayout/EditPageLayout.i18n.php

```diff
@@ -157,6 +153,14 @@
 	'prefs-starting-an-edit' => 'This is a heading in Special:Preferences',
 	'prefs-editing-experience' => 'This is a heading in Special:Preferences',
 	'wikia-editor-preview-editSummary' => 'A text showed on the bottom of preview window when the edit summary was added, the parameter is the added summary',
+
+	'editpagelayout-preview-label-desktop' => "A label shown under the desktop icon in the preview section of the edit interface, clicking changes the type of preview (toggle between mobile and desktop). Also displayed as a popup when the user hovers the desktop icon",
+	'editpagelayout-preview-label-mobile' => "A label shown under the mobile icon in the preview section of the edit interface, clicking changes the type of preview (toggle between mobile and desktop). Also displayed as a popup when the user hovers over the mobile icon",
+
+	'editpagelayout-preview-dropdown-item-mobile' => "An item in a dropdown menu near the top of the preview modal dialog, allows the user to select the type of preview they want to see",
+	'editpagelayout-preview-dropdown-item-tablet' => "An item in a dropdown menu near the top of the preview modal dialog, allows the user to select the type of preview they want to see",
+	'editpagelayout-preview-dropdown-item-desktop' => "An item in a dropdown menu near the top of the preview modal dialog, allows the user to select the type of preview they want to see",
+	'editpagelayout-preview-dropdown-item-desktopxl' => "An item in a dropdown menu near the top of the preview modal dialog, allows the user to select the type of preview they want to see"
```

Comment:
If the string doesn't already have single quotes in it, prefer single quotes per coding conventions.